Cargo.toml

```diff
@@ -19,9 +19,10 @@ categories = ["data-structures", "no-std"]
 
 [dependencies]
 serde = { version = "1.0", optional = true, features = ["derive"] }
+borsh = { version = "1.5.4", optional = true, features = ["derive"] } # using 1.5.4 as it is good regarding syn
```

Comment:
Does borsh have an MSRV policy? Does it have a policy on how often it makes breaking changes?

Reply:
I use proc macro, as per changelog it seems for proc macro they need 1.67

Reply:
made `>=1.5.4, <1.6`. 1.5.3 was weirdly broken for some combination of features.

Reply:
the test only
```
  tests:
    strategy:
      matrix:
        rust_version: [1.67, stable]
```

without macro, they may be do earlier MSRV, but I found no evidence for that

Reply:
I can do without macro easy if needed.

Reply:
Given the syn issues, I'd prefer a manual impl.